template/build.gradle.kts

```diff
@@ -54,19 +59,26 @@ detekt {
 
     ignoredBuildTypes = listOf("release")
     ignoredFlavors = listOf("production")
+}
 
+tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
+    jvmTarget = JavaVersion.VERSION_11.toString()
     reports {
+        xml.required.set(true)
+        html.required.set(true)
         xml {
-            enabled = true
-            destination = file("build/reports/detekt.xml")
+            outputLocation.set(file("build/reports/detekt.xml"))
         }
         html {
-            enabled = true
-            destination = file("build/reports/detekt.html")
+            outputLocation.set(file("build/reports/detekt.html"))
         }
     }
 }
 
+tasks.withType<io.gitlab.arturbosch.detekt.DetektCreateBaselineTask>().configureEach {
```

Comment:
Any idea why we need this? 🙏

Reply:
@Tuubz They're redundant, we only need `tasks.withType<io.gitlab.arturbosch.detekt.Detekt>` 🙏